src/web/shared/Styles.ts

```diff
@@ -507,12 +509,25 @@ export function divePacketIconStyle(feature: Feature, animatedColor?: string) {
     })
 }
 
-export function driftPacketIconStyle(feature: Feature, animatedColor?: string) {
+/**
+ * Returns the drift icon index that should be displayed, given a drift speed.
+ * 
+ * @param driftSpeed Speed of the drift, in m/s
+ * @returns Index into Styles.driftArrowPngs, of the icon sthat should represent this drift
+ */
+export function driftSpeedToBinIndex(driftSpeed: number) {
     // 6 bins for drift speeds of 0 m/s to 2.5+ m/s
-    // Bin numbers correspond with the number of tick marks on the drift arrow visually indicating the speed of the drift to the operator
-    const maxBins = 6
+    // Bin numbers (+ 1) correspond with the number of tick marks on the drift arrow visually indicating the speed of the drift to the operator
+    if (driftSpeed == null) return 0
+    
     const binValueIncrement = 0.5
-    let binNumber = Math.floor(feature.get('speed') ?? 0 / binValueIncrement)
+    return Math.floor(driftSpeed / binValueIncrement)
+}
+
+
+export function driftPacketIconStyle(feature: Feature, animatedColor?: string) {
```

Comment:
Can we add the a comment before this function like the driftspeedtobinindex function  /***/

Reply:
https://github.com/jaiarobotics/jaiabot/blob/1.y/src/doc/markdown/page_110_coding_standards.md